grype/presenter/cyclonedx/presenter_test.go

```diff
@@ -3,101 +3,25 @@ package cyclonedx
 import (
 	"bytes"
 	"flag"
-	"regexp"
 	"testing"
 
 	"github.com/sergi/go-diff/diffmatchpatch"
 
 	"github.com/anchore/go-testutils"
-	"github.com/anchore/grype/grype/match"
-	"github.com/anchore/grype/grype/pkg"
 	"github.com/anchore/grype/grype/presenter/models"
-	"github.com/anchore/grype/grype/vulnerability"
-	"github.com/anchore/stereoscope/pkg/imagetest"
-	syftPkg "github.com/anchore/syft/syft/pkg"
 	"github.com/anchore/syft/syft/source"
 )
 
 var update = flag.Bool("update", false, "update the *.golden files for cyclonedx presenters")
 
-func createResults() (match.Matches, []pkg.Package) {
-
-	pkg1 := pkg.Package{
-		ID:      "package-1-id",
-		Name:    "package-1",
-		Version: "1.0.1",
-		Type:    syftPkg.DebPkg,
-	}
-	pkg2 := pkg.Package{
-		ID:      "package-2-id",
-		Name:    "package-2",
-		Version: "2.0.1",
-		Type:    syftPkg.DebPkg,
-		Licenses: []string{
-			"MIT",
-			"Apache-v2",
-		},
-	}
-
-	var match1 = match.Match{
-
-		Vulnerability: vulnerability.Vulnerability{
-			ID:        "CVE-1999-0001",
-			Namespace: "source-1",
-		},
-		Package: pkg1,
-		Details: []match.Detail{
-			{
-				Type:    match.ExactDirectMatch,
-				Matcher: match.DpkgMatcher,
-			},
-		},
-	}
-
-	var match2 = match.Match{
-
-		Vulnerability: vulnerability.Vulnerability{
-			ID:        "CVE-1999-0002",
-			Namespace: "source-2",
-		},
-		Package: pkg2,
-		Details: []match.Detail{
-			{
-				Type:    match.ExactIndirectMatch,
-				Matcher: match.DpkgMatcher,
-				SearchedBy: map[string]interface{}{
-					"some": "key",
-				},
-			},
-		},
-	}
-
-	matches := match.NewMatches()
-
-	matches.Add(match1, match2)
-
-	return matches, []pkg.Package{pkg1, pkg2}
-}
-
 func TestCycloneDxPresenterImage(t *testing.T) {
 	var buffer bytes.Buffer
 
-	matches, packages := createResults()
-
-	img := imagetest.GetFixtureImage(t, "docker-archive", "image-simple")
-	s, err := source.NewFromImage(img, "user-input")
```

Comment:
This was where we would invoke docker in certain executions to build the image and generate the source. 969 asks that anything that the unit level be dependency agnostic which is the primary driving force behind this refactor